src/test/java/com/bobocode/hoverla/bring/BringApplicationTest.java

```diff
@@ -0,0 +1,71 @@
+package com.bobocode.hoverla.bring;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import com.bobocode.hoverla.bring.context.ApplicationContext;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class BringApplicationTest {
+    private final String PACKAGE = "com.bobocode.hoverla.bring";
+
+    @DisplayName("Throws IllegalArgumentException: no packages were provided")
+    @Test
+    void throwsExceptionPackageToScanIsEmpty() {
```

Comment:
Basically, you should check it with null as well. You may create a simple parametrized test for it.

Reply:
@DanielChanger chaged. Thank you for an advice and guide link!